src/components/sidebar/components/sidebar-back-to-link/index.tsx

```diff
@@ -1,12 +1,16 @@
 import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import MaybeInternalLink from 'components/maybe-internal-link'
 import s from './style.module.css'
 
 // TODO: design is still planning this functionality & states
-const SidebarBackToLink: React.FC = () => (
-  <a className={s.backToLink}>
+const SidebarBackToLink: React.FC<{
+  text?: string
+  url?: string
+}> = ({ text = 'Back to lorem ipsum', url = '/' }) => (
+  <MaybeInternalLink className={s.backToLink} href={url}>
     <IconArrowLeft16 className={s.icon} />
-    <span>Back to lorem ipsum</span>
-  </a>
+    <span>{text}</span>
+  </MaybeInternalLink>
 )
```

Comment:
It was decided last week via Slack to use a placeholder for this component, I've implemented it in my sidebar PR here: https://github.com/hashicorp/dev-portal/pull/58/files#diff-969313d21e13fab2f5de2cb624c051abb51b3ab45018f0bedff88df293fbf37a

Reply:
Ah okay - most recent feedback I have from design is to do `← Developer` on `/waypoint` and `← Waypoint` on `/waypoint/docs`. Will wait till #58 merges, and will double-check with design what they'd like to demo before merging this PR 👍

Reply:
Here's the latest thread I have: https://hashicorp.slack.com/archives/C028LTF7TJ5/p1640279174192700?thread_ts=1640273232.181900&cid=C028LTF7TJ5. This is definitely confusing! Can we try to elicit that feedback in `#proj-dev-portal-eng` that way everyone will be aware of the latest? I'm also having a working session later today with Brayden where I'll be getting some QA feedback, I'll make sure we what we cover documented somewhere like Slack or Asana or a Google doc.

Reply:
I went back to that Slack thread to ask for clarification for what we want to show tomorrow, and the vote is for showing a placeholder for now: https://hashicorp.slack.com/archives/C028LTF7TJ5/p1641330820226500?thread_ts=1640273232.181900&cid=C028LTF7TJ5